prow/config.yaml

```diff
@@ -90,8 +128,128 @@ tide:
+            auto: false
+          highlight_regexes:
+          - Automatic merge failed
+          - cannot convert.+to type
+          - "cannot use.+as.+(type|value)"
+          - Cluster failed to initialize
+          - cluster unreachable
+          - "compilepkg: error"
+          - configuration error
+          - contact Google support
+          - could not apply prowjob annotations
+          - could not find the referenced.+TestGroup
+          - could not write config
+          - "couldn't load prow config:"
+          - curl.+Failed to connect
+          - DATA RACE
+          - dirty working directory
+          - ^E\d{4} \d\d:\d\d:\d\d\.\d+
+          - "(Error|ERROR|error)s?:"
+          - Error.+executing benchmarks
+          - Expected.+got
+          - FAILED. See logs
+          - failed to acquire k8s binaries
+          - failed to solve
+          - (FAIL|Failure \[)\b
+          - fatal error
+          - flag provided but not defined
+          - Full Stack Trace
+          - got.+expected
+          - hash mismatch
+          - Hit an unsupported type
+          - imported but not used
+          - Incompatible changes
+          - incorrect boilerplate
+          - indent-error-flow
+          - INSTALLATION FAILED
+          - is a misspelling
+          - LimitExceeded
+          - "make:.+Error"
+          - Master not detected
+          - Merge conflict
+          - not enough arguments in call
+          - panic\b
+          - Previous (write|read)
+          - Process did not finish before.+timeout
+          - race detected
+          - security token.+invalid
+          - "signal: killed"
+          - Something went wrong
+          - too few arguments
+          - too many errors
+          - "[Tt]imed out"
+          - type.+has no field
+          - unable to start the controlplane
+          - "undefined:"
+          - Unfortunately, an error
+          - unused.+deadcode
+          - "[Uu]nexpected error"
+          - verify.+failed
+          - want.+got
+          - Your cluster may not be fully functional
+          - ^\s+\^$
+          - \[(0;)?31m
+          - "^diff " # [+-]{3}\\s" has too much noise from go test output and set -x
+          - • Failure
+          # This highlights the start of bazel tests/runs to skip go importing noise.
+          - "^INFO: Analyzed \\d+ targets"
+      required_files:
+        - ^.*build-log\.txt$
+    - lens:
+        name: junit
+      required_files:
+        - ^artifacts(/.*/|/)junit.*\.xml$ # https://regex101.com/r/vCSegS/1
+    - lens:
+        name: coverage
+      required_files:
+        - ^artifacts/filtered\.cov$
+      optional_files:
+        - ^artifacts/filtered\.html$
+    - lens:
+        name: podinfo
+        config:
+          runner_configs:
+            "default":
+              pod_link_template: "https://console.cloud.google.com/kubernetes/pod/us-central1-f/prow/test-pods/{{ .Name }}/details?project=k8s-prow-builds"
```

Comment:
These links aren't valid for us, obviously, but I'm assuming you just haven't configured spyglass at all yet?

Reply:
Good catch, I did configure some of it, so all the top links work, but I had missed these :)